Marlin/src/pins/stm32f4/pins_OPULO_LUMEN_REV4.h

```diff
@@ -204,3 +204,6 @@
 #define LUMEN_AUX3_PWM2                     PB9
 #define LUMEN_AUX3_A1                       PA0
 #define LUMEN_AUX3_A2                       PA1
+
+#define RS485_TX_ENABLE_PIN                 PD11
+#define RS485_RX_ENABLE_PIN                 PD12
```

Comment:
Another 'burning question'…. Is a board required to have RS485-specific hardware, with specific pins assigned to it? If that is the case, then we can sanity-check the `RS485_SERIAL` port number and make sure it corresponds to the required hardware port.

Reply:
Just my 2c. This is the first time I see a RX enable pin on 485 serial communications, isn't it a resources waste?
RS485 is half duplex then you are transmitting you may not receive, then a master to slave protocol should be implemented, then to correclty handle communications answers, just purge RX buffer after TX.
I'm wrong or am I missing something ?

Reply:
> Is a board required to have RS485-specific hardware, with specific pins assigned to it?

Yep!

> If that is the case, then we can sanity-check the RS485_SERIAL port number and make sure it corresponds to the required hardware port.

I'm not sure how to do that and my need your help.

@GMagician "resource waste" is always debatable. The code this PR uses already has a controller/peripheral protocol. Our long term goal is to make it so not even that is required and we can do cool things like having an autoreporter automatically report up to the host when there's a valid RS485 message, even if that isn't a response to an M485 command. So we're fine with the "waste" as it is now.